package.json

```diff
@@ -18,15 +18,16 @@
     "prettier-format": "prettier --write \"**/*.{js,ts,tsx,json,scss,html}\"",
     "start": "(test -f client/build/index.html || yarn build:client) && (test -f ssr/dist/main.js || yarn build:ssr) && (test -d client/build/en-us/_spas || yarn build --spas-only) && nf -j Procfile.start start",
     "start:client": "cd client && cross-env BROWSER=none PORT=3000 react-scripts start",
-    "start:dev-server": "nodemon server --watch server --watch build --watch content --watch kumascript --ext js",
+    "start:dev-server": "nodemon server --watch server --watch build --watch content --watch kumascript --ext js --watch ssr/dist",
```

Comment:
Should it just be `ssr` like the others?

Reply:
No because editing the `ssr/*.js` should not trigger a nodemon restart. But editing the `ssr/dist/*.js` should.
So if I "heed your suggestion", it would trigger two `nodemon` restarts on 1 save.

Reply:
Makes sense, I was just wondering about https://github.com/mdn/yari/blob/master/ssr/mozilla.dnthelper.min.js

Reply:
Fair point but that one is weird enough as it is. Don't think anybody ever edits or understands how it works. 
I bet we're not doing GA the bestest possible way. Just struggling to find the energy to deep investigate it.